doc/source/reference/arrays.nditer.rst

```diff
@@ -220,20 +220,37 @@ produce identical results to the ones in the previous section.
     ...     print("%d <%d>" % (it[0], it.index), end=' ')
     ...     it.iternext()
     ...
-    0 <0> 1 <2> 2 <4> 3 <1> 4 <3> 5 <5>
+    0 <0> True
+    1 <2> True
+    2 <4> True
+    3 <1> True
+    4 <3> True
+    5 <5> False
```

Comment:
This doesn't look right - what is printing the booleans?

Reply:
In a IPython terminal, I obtain the "old" result. When running the `refguide_check.py` tool the output is as in the commit.

Reply:
In a "plain" Python REPL, the output contains the booleans. All lines that return a value seem to appear. This type of detail is hard to find in the Python docs, it seems that it is "assumed to be known"...